pandas/sparse/frame.py

```diff
@@ -302,7 +302,28 @@ def fillna(self, value=None, method=None, axis=0, inplace=False,
     # ----------------------------------------------------------------------
     # Support different internal representation of SparseDataFrame
 
-    def _sanitize_column(self, key, value):
+    def _sanitize_column(self, key, value, broadcast=True):
+        """
+        Creates a new SparseArray from the input value.
+
+        Parameters
+        ----------
+        keys : object
+        value : int, Series, or array-like
+        broadcast : bool
+            Indicates whether all columns with the given key should be
+            returned.
+
+        Returns
+        -------
+        sanitized_column : SparseArray
+
+        Notes
+        -----
+        The "broadcast" parameter was added to match the method signature of
```

Comment:
Just to be clear, you want **kwargs for SparseDataFrame._sanitize_column and an optional parameter for DataFrame._sanitize_column? Should I add *args too so there's no risk of error?

``key`` isn't used in SparseDataFrame._sanitize_column either. Do you want me to get rid of that, too?